x-pack/solutions/security/packages/upselling/sections/attack_discovery/index.tsx

```diff
@@ -30,7 +30,7 @@ const AttackDiscoveryUpsellingSectionComponent: React.FC<Props> = ({
     () => (
       <EuiFlexGroup alignItems="center" direction="column" gutterSize="none">
         <EuiFlexItem data-test-subj="assistantAvatar" grow={false}>
-          <AssistantAvatar size="m" />
+          <AssistantBeacon />
```

Comment:
I downgraded from Enterprise to Basic and observed an unexpected background behind the `AssistantBeacon` on the Attack discovery upselling page, as illustrated by the following screenshot:

![attack_discovery_upsell_pr](https://github.com/user-attachments/assets/f01f1dc7-9d00-43f5-acfc-2fcf00d4b40d)

Reply:
There appears to be a similar unexpected background in light mode, as illustrated by the screenshot below:

![light_mode](https://github.com/user-attachments/assets/dc9f35a4-c279-4dc5-9bfc-6ad6e56e01ef)